### PR TITLE
Port to modelforge 0.11

### PR DESCRIPTION
**Pull request by warenlg**
Signed-off-by: Waren Long <waren@sourced.tech>

### CODE REVIEW

**Comment by vmarkovtsev** [comment]
@warenlg `series` is described in https://docs.sourced.tech/modelforge/model#series

File: sourced/ml/cmd/bigartm2asdf.py [hunk 1]
```diff
@@ -39,4 +39,4 @@ def bigartm2asdf(args):
     indices = numpy.array(indices, dtype=numpy.int32)
     matrix = csr_matrix((data, indices, indptr), shape=(len(tokens), len(items) - 2)).T
     Topics().construct(tokens=tokens, topics=None, matrix=matrix) \
-        .save(output=args.output, series="topics")
+        .save(output=args.output, series="bigartm")
```
Comment (vmarkovtsev) [inline sourced/ml/cmd/bigartm2asdf.py]: so the `series` should correspond to the problem the model is solving. In our case, this is topic modeling, so we should leave "topics"

File: sourced/ml/cmd/id2vec_postprocess.py [hunk 2]
```diff
@@ -40,4 +40,5 @@ def id2vec_postprocess(args):
     log.info("Generating numpy arrays...")
     embeddings = numpy.array(embeddings, dtype=numpy.float32)
     log.info("Writing %s...", args.output)
-    Id2Vec().construct(embeddings=embeddings, tokens=tokens).save(args.output)
+    Id2Vec().construct(embeddings=embeddings, tokens=tokens) \
+        .save(output=args.output, series="swivel-data")
```
Comment (vmarkovtsev) [inline sourced/ml/cmd/id2vec_postprocess.py]: swivel-data -> id2vec

File: sourced/ml/models/bow.py [hunk 3]
```diff
@@ -84,7 +84,7 @@ def __len__(self):
         """
         return len(self._documents)
 
-    def save(self, output: str, series: str = "bow", deps: Iterable = tuple(),
```
Comment (vmarkovtsev) [inline sourced/ml/models/bow.py]: The BOW model is very generic, we cannot provide any reasonable defaults

File: sourced/ml/cmd/merge_coocc.py [hunk 4]
```diff
@@ -115,4 +115,4 @@ def merge_coocc_no_spark(df, filepaths, log, args):
             result.data[indx_overflow] = MAX_INT32
     Cooccurrences() \
         .construct(df.tokens(), result[:-1, :-1]) \
-        .save(args.output, (df,))
+        .save(output=args.output, series="merge-cooc-no-spark", deps=(df,))
```
Comment (vmarkovtsev) [inline sourced/ml/cmd/merge_coocc.py]: merge-cooc-no-spark -> id2vec (if I am not mistaken; depends on the problem we are solving with this command)

File: sourced/ml/models/coocc.py [hunk 5]
```diff
@@ -10,6 +10,8 @@ class Cooccurrences(Model):
     Co-occurrence matrix.
     """
     NAME = "co-occurrences"
+    VENDOR = "source{d}"
+    DESCRIPTION = "Model that contains the sparse co-occurrence matrix of source code identifiers."
```
Comment (vmarkovtsev) [inline sourced/ml/models/coocc.py]: Let's include the license to all our models

File: requirements.txt [hunk 6]
```diff
@@ -1,7 +1,7 @@
 Cython>=0.28,<1.0; python_version == '3.7'
 PyStemmer==1.3.0
 bblfsh==2.12.7
-modelforge==0.9.3
+modelforge==0.10.1
```
Comment (EgorBu) [inline requirements.txt]: `modelforge 0.11`?

File: setup.py [hunk 7]
```diff
@@ -33,7 +33,7 @@
     install_requires=[
         "PyStemmer>=1.3,<2.0",
         "bblfsh>=2.2.1,<3.0",
-        "modelforge>=0.9.3,<0.10",
+        "modelforge>=0.10.1,<0.11",
```
Comment (EgorBu) [inline setup.py]: `modelforge 0.11` is not included - is it correct?

Comment (warenlg) [reply setup.py]: Oh modelforge 0.11 was just relesed yesterday, let me update it

**Pull request merged**